clients/src/main/java/org/apache/kafka/clients/admin/KafkaAdminClient.java

```diff
@@ -3178,7 +3178,8 @@ public DescribeConsumerGroupsResult describeConsumerGroups(final Collection<Stri
                 DescribeConsumerGroupsHandler.newFuture(groupIds);
         DescribeConsumerGroupsHandler handler = new DescribeConsumerGroupsHandler(options.includeAuthorizedOperations(), logContext);
         invokeDriver(handler, future, options.timeoutMs);
-        return new DescribeConsumerGroupsResult(future.all());
+        return new DescribeConsumerGroupsResult(future.all().entrySet().stream()
```

Comment:
A bit annoying that we have to do this, but this will have to do for now.

Reply:
Indeed, I felt it wasn't worth trying to refactor the `DescribeConsumerGroupsHandler` at this point.